deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -553,39 +553,42 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
             object: nil,
             queue: OperationQueue.main
         ) { [weak self] notification in
-            guard let self, let id = notification.userInfo?["message_id"] as? Int else { return }
-            if self.dcChat.canSend, id > 0 {
-                let msg = self.dcContext.getMessage(id: id)
-                if msg.isInfo,
-                   let parent = msg.parent,
-                   parent.type == DC_MSG_WEBXDC {
-                    self.refreshMessages()
-                } else {
-                    self.updateMessage(msg)
+            guard let self, let ui = notification.userInfo else { return }
+            let chatId = ui["chat_id"] as? Int ?? 0
```

Comment:
What's so special about the `chatId 0`?

Reply:
in theory, these are batch updates, affecting multiple chats. i assume, they're not really in use in practise, but i did not check and mimiced android-behavior